### ISSUE TITLE
Configuration API

### ISSUE THREAD

**Issue by ImagineWave** [AuthMe/AuthMeReloaded#2871]
### What feature do you want to see added?

Override configuration settings through API

for example I need to override login/register message interval

### Are there any alternatives?

Right now it's possible only though reflection dark magic

**Comment by sgdc3** [comment]
You could use reflections to obtain the AuthMe.injector instance and then invoke injector.getSingleton(SettingsManager) to interact with our settings manager.

**Comment by ImagineWave** [comment]
How to set Property<Integer> MESSAGE_INTERVAL? I should use ur ch.jalu.configme.properties.Property ?

**Comment by ljacqu** [comment]
The properties’ values can be changed in config.yml

**Comment by ImagineWave** [comment]
> The properties’ values can be changed in config.yml

BRUH
are u suggesting as a solution that I corrupt user configs?
I'm trying to do integration plugin

**Comment by ljacqu** [comment]
I don't think you should override settings that a user can change in a config file. But what sgdc3 said would indeed work.
